### PR TITLE
add home base

### PR DESCRIPTION
**Pull request by kamikaz1k**
- added home base sprite to game scene
- factored out the tile slicing stuff to be cleaner
- added GameOver screen
- added tint for base hit, and bad dead

### Demo
https://user-images.githubusercontent.com/11997716/201150421-cc53ebab-1503-441c-8141-65f41687f5d1.mov

### CODE REVIEW

File: src/entities/HomeBase.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    if (this.hp <= 0) {
+      this.hp = 0;
+    }
+  }
+
+  update(time: number, delta: number) {
+    this.healthBar.setHealth(this.hp).draw();
+
+    if (this.hp <= 0) {
+      this.setTint(0x0f0f0f);
```
Comment (nafeu) [inline src/entities/HomeBase.ts:52]: This is minor but for things like tint colors, try and store in a constants file

**Review by nafeu — APPROVED** [review]
Looks good man, there a bunch of magic numbers floating around with regards to the Game Over scene, but those don't get in the way of gameplay logic implementation so thats okay for now

**Comment by kamikaz1k** [comment]
@nafeu yea it's just layout/cosmetic stuff, it'll depend on final screen dimensions / design etc. IMHO hiding it in a config is premature, but we can chat about preference for this next chat if you want?

File: src/entities/HomeBase.ts (hunk 1 above)
Comment (kamikaz1k) [reply src/entities/HomeBase.ts]: fixed!

**Pull request merged**